integration-tests/src/test/java/oracle/kubernetes/operator/ItElasticLogging.java

```diff
@@ -318,4 +389,96 @@ private String execElasticStackQuery(String queryCriteria) throws Exception {
 
     return result.stdout();
   }
+  
+  private void downloadWLSLoggingExporterJars() throws Exception {
+    // Location of Weblogic logging exporter jar files
+    final String loggingJarRepos = 
+      "https://github.com/oracle/weblogic-logging-exporter/releases/download/v0.1";
```

Comment:
v0.1.1 is available now

Reply:
changed to:

  private final String loggingJarRepos = 
    "https://github.com/oracle/weblogic-logging-exporter/releases/download/v0.1.1";
  private final String wlsLoggingExpJar = "weblogic-logging-exporter-0.1.1.jar";

and the test downloaded it successfully